app/src/main/res/values/theme.xml

```diff
@@ -63,4 +63,20 @@
     <attr name="colorButton" format="reference" />
     <attr name="textColorButton" format="reference" />
 
+    <style name="Hedvig.Theme.Chat" parent="Hedvig.Theme">
+        <item name="android:colorBackground">@color/colorBackgroundChat</item>
```

Comment:
This seems suspect. Is the background color of the chat inconsistent with the rest of the application?

Reply:
Yes, the dark background on the chat screen is #1B1B1B and the others are #121212. Maybe I should talk to Zak about that

Reply:
Yes, go 👮